src/bio/bio_monitor.c

```diff
@@ -821,6 +823,20 @@ bio_fini_health_monitoring(struct bio_blobstore *bb)
 		spdk_bdev_close(bdh->bdh_desc);
 		bdh->bdh_desc = NULL;
 	}
+
+	/*
+	 * Init xstream will finialize bdev subsystem later, so we need
+	 * to wait for the inflight health collecting request done.
+	 */
+	D_ASSERT(bdh->bdh_inflights == 0 || bdh->bdh_inflights == 1);
```

Comment:
```suggestion
        D_ASSERT(bdh->bdh_inflights < 2);
```
as bdh_inflights is an unsigned int